### PR TITLE
add new unit tests for Kube methods

### PR DESCRIPTION
**Pull request by zreigz**
## Summary
add new unit tests for Kube methods

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: cmd/plural/proxy_test.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import (
+	"os"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/pluralsh/plural-operator/api/platform/v1alpha1"
+	"github.com/stretchr/testify/mock"
+
+	plural "github.com/pluralsh/plural/cmd/plural"
```
Comment (maciaszczykm) [inline cmd/plural/proxy_test.go:12]: `plural` prefix can be removed.

File: cmd/plural/ops_test.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"gopkg.in/yaml.v2"
+
+	plural "github.com/pluralsh/plural/cmd/plural"
```
Comment (maciaszczykm) [inline cmd/plural/ops_test.go:12]: `plural` prefix can be removed.

File: cmd/plural/ops_test.go [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+	"github.com/pluralsh/plural/pkg/manifest"
+	"github.com/pluralsh/plural/pkg/test/mocks"
+	"github.com/pluralsh/plural/pkg/utils/git"
+
+	v1 "k8s.io/api/core/v1"
```
Comment (maciaszczykm) [inline cmd/plural/ops_test.go:17]: `v1` prefix can be removed.

**Review by maciaszczykm — COMMENTED** [review]
Looks good, I've left few nit-picks.

File: cmd/plural/ops_test.go (hunk 2 above)
Comment (zreigz) [reply cmd/plural/ops_test.go:12]: strange but code can't be compiled without this prefix

File: cmd/plural/ops_test.go (hunk 3 above)
Comment (zreigz) [reply cmd/plural/ops_test.go:17]: linter requires this

**Pull request merged**